tests/data/ciscosb_cbs350.json

```diff
@@ -0,0 +1,7571 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.9.6.1.1004.28.5",
+                    "sysDescr": "CBS350-24P-4G 24-Port Gigabit PoE Managed Switch",
+                    "sysContact": "<private>",
+                    "version": "",
```

Comment:
Should extract this and hardware while at it!

Reply:
Right ! done.